py3status/modules/scratchpad_async.py

```diff
@@ -37,6 +35,17 @@ class Meta:
                     'msg': '{} should not be used in format use `{counter}`',
                 },
             ],
+            'substitute_by_value': [
+                {
+                    'param': 'always_show',
+                    'value': True,
+                    'substitute': {
+                        'param': 'format',
+                        'value': '{counter} ⌫',
+                    },
+                    'msg': 'obsolete parameter. update format instead',
+                },
```

Comment:
If we omit this `substitute_by_value`, then we took the users format into account... only if they have one. Same thing works with with default `always_show = False`... only if they have one.
What about those with both configs? 

I think we remove all deprecation and merge in. 😇